src/pages/components/button/style.mdx

```diff
@@ -6,207 +6,435 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>Sizes</AnchorLink>
```

Comment:
```suggestion
<AnchorLink>Size</AnchorLink>
```

The content template says that this heading is not plural, so following it for consistency.